crates/sui-graphql-rpc/src/types/transaction_block/mod.rs

```diff
@@ -497,6 +556,63 @@ impl Loader<DigestKey> for Db {
     }
 }
 
+#[async_trait::async_trait]
+impl Loader<SeqKey> for Db {
+    type Value = TransactionBlock;
+    type Error = Error;
+
+    async fn load(&self, keys: &[SeqKey]) -> Result<HashMap<SeqKey, TransactionBlock>, Error> {
+        use transactions::dsl as tx;
+
+        let seqs: Vec<_> = keys.iter().map(|k| k.tx_sequence_number as i64).collect();
```

Comment:
As per Emma's investigation, we don't need to do that here, because we are using an `eq_any` query, so it will correctly return an empty result if there are no keys.